Sleep Tracker/SleepTracker/View/SleepSetupView/Subviews/NoAlarmLabelView.swift

```diff
@@ -0,0 +1,20 @@
+import SwiftUI
+
+/// A view that shows information about the disabled alarm clock.
+struct NoAlarmLabelView: View {
+    var body: some View {
+        Text("No alarm clock,\njust a sleep analysis")
```

Comment:
nit:

Do we need a comma?
```suggestion
        Text("No alarm clock\njust a sleep analysis")
```

Reply:
fixed